src/patternfly/components/InputGroup/styles.scss

```diff
@@ -62,6 +77,32 @@
     background-color: var(--pf-c-input-group__text--BackgroundColor);
     border: var(--pf-c-input-group__text--BorderSize) solid var(--pf-c-input-group__text--BorderColor);
   }
+  &__action {
+    position: absolute;
+    padding: var(--pf-c-input-group__action--PaddingTop) var(--pf-c-input-group__action--PaddingRight) var(--pf-c-input-group__action--PaddingBottom) var(--pf-c-input-group__action--PaddingLeft);
+    font-size: var(--pf-c-input-group__action--FontSize);
+    color: var(--pf-c-input-group__action--Color);
+    cursor: pointer;
+    border: none;
+
+    &:hover{
+      color: var(--pf-c-input-group__action-hover--color);
+    }
+
+    &.pf-m-right {
+      right: 0;
```

Comment:
`.pf-m-align-right` and `.pf-m-align-left` would probably work better. Right now `.pf-m-top/bottom/left/right` are used to display something to the top/bottom/left/right, like a tooltip or popover or a "dropup" dropdown (uses `.pf-m-top`).

`.pf-m-alight-right/left` aligns something to the left or right edge of the parent.

Reply:
👍